### ISSUE TITLE
libgcc_ng and libstdcxx_ng listed as requirements

### ISSUE THREAD

**Issue by jakirkham** [regro/conda-press#52]
When running `conda press` on the `numcodecs` packages recently using `--fatten --skip-python`, it appears that `libgcc_ng` and `libstdcxx_ng` were listed as requirements and their libraries not added. As a result `pip` fails to install as these libraries are missing. Additionally the wheels don't have these standard compiler libraries they are linked against.

**Comment by scopatz** [comment]
Thanks for reporting! can you please provide the full command you ran?

**Comment by jakirkham** [comment]
Sure, I ran `conda press --skip-python --fatten` on all of the tarballs for the `numcodecs` version `0.6.4` release.

Used [this script]( https://gist.github.com/jakirkham/7baa42250a0bddf0e2eb5e7db3dca1bc ) to download them and run `conda press` on each one.

Should add I'm new to `conda-press`. So could be doing foolish things. Guidance is also appreciated. 🙂

**Comment by jakirkham** [comment]
What happens when you run `conda press` on a tarball? Is the code path different than for other things (like the package spec)?

**Comment by scopatz** [comment]
> What happens when you run conda press on a tarball? Is the code path different than for other things (like the package spec)?

Basically the only difference is that when running on a tarball it doesn't do a solve, and so doesn't build the wheels for the deps as well. It only builds the wheel for that tarball.  Very open to suggestions for changes here!

**Comment by jakirkham** [comment]
Yeah it would be nice if it still solved with tarballs. My goal was to generate a wheel from a specific Conda package. So using a tarball helped facilitate that.

**Comment by scopatz** [comment]
Ok great!
